charts/athens-proxy/README.md

```diff
@@ -81,3 +170,11 @@ sshGitServers:
     ## ssh port
     port: 22
 ```
+
+## Testing
+
+Using `chart-testing` to lint, install and test the chart on a local Kubernetes (Minikube, Rancher Desktop, ...)
```

Comment:
```suggestion
Run `chart-testing` to lint, install and test the chart on a local Kubernetes (Minikube, Rancher Desktop etc.)
```

Reply:
Also not sure it's necessary to specify Minikube/Docker Desktop/any examples at all